src/Spatial/Euclidean/Quaternion.cs

```diff
@@ -166,43 +136,46 @@ public bool IsRotation
         public EulerAngles ToEulerAngles()
         {
             return new EulerAngles(
-                Angle.FromRadians(Math.Atan2(2*(_w*_x + _y*_z), ((_w*_w) + (_z*_z) - (_x*_x) - (_y*_y)))),
-                Angle.FromRadians(Math.Asin(2*(_w*_y - _x*_z))),
-                Angle.FromRadians(Math.Atan2(2*(_w*_z + _x*_y), ((_w*_w) + (_x*_x) - (_y*_y) - (_z*_z)))));
+                Angle.FromRadians(Math.Atan2(2 * (_w * _x + _y * _z), ((_w * _w) + (_z * _z) - (_x * _x) - (_y * _y)))),
+                Angle.FromRadians(Math.Asin(2 * (_w * _y - _x * _z))),
+                Angle.FromRadians(Math.Atan2(2 * (_w * _z + _x * _y), ((_w * _w) + (_x * _x) - (_y * _y) - (_z * _z)))));
         }
 
         /// <summary>
         /// Returns a new Quaternion q with the Scalar part only.
         /// If you need a Double, use the Real-Field instead.
         /// </summary>
-        public Quaternion Scalar()
+        public Quaternion Scalar
         {
-            return new Quaternion(_w, 0, 0, 0);
+            get { return new Quaternion(_w, 0, 0, 0); }
         }
 
         /// <summary>
         /// Returns a new Quaternion q with the Vector part only.
         /// </summary>
-        public Quaternion Vector()
+        public Quaternion Vector
         {
-            return new Quaternion(0, _x, _y, _z);
+            get { return new Quaternion(0, _x, _y, _z); }
         }
 
         /// <summary>
         /// Returns a new normalized Quaternion u with the Vector part only, such that ||u|| = 1.
-        /// Q may then be represented as q = r*(cos(phi) + u * sin(phi)) = r*exp(phi*u) where r is the absolute and phi the argument of q.
+        /// Q may then be represented as q = r*(cos(phi) + u*sin(phi)) = r*exp(phi*u) where r is the absolute and phi the argument of q.
         /// </summary>
-        public Quaternion UnitVector()
+        public Quaternion NormalizedVector
         {
-            return ToUnitQuaternion(0, _x, _y, _z);
+            get { return ToUnitQuaternion(0, _x, _y, _z); }
         }
 
         /// <summary>
         /// Returns a new normalized Quaternion q with the direction of this quaternion.
         /// </summary>
-        public Quaternion Sign()
+        public Quaternion Normalized
```

Comment:
For Normailze and Invert i would use property because 
quaternion.Normalize() suggests (for me) that I am going to normalize quaternion not return a new one. Whereas property says "I am going to take quaternion and return normalized version of it" 
For me object method suggest that we are going to change this object. But yeah, for consistency I am going to change it.